core/src/wallet/ripple/api_impl/RippleLikeTransactionApi.cpp

```diff
@@ -351,12 +358,21 @@ namespace ledger {
             return *this;
         }
 
+        RippleLikeTransactionApi &RippleLikeTransactionApi::setDestinationTag(uint32_t tag) {
+            _destinationTag = tag;
+            return *this;
+        }
+
         std::vector<api::RippleLikeMemo> RippleLikeTransactionApi::getMemos() {
             return _memos;
         }
 
         void RippleLikeTransactionApi::addMemo(api::RippleLikeMemo const& memo) {
             _memos.push_back(memo);
         }
+
+        std::experimental::optional<int64_t> RippleLikeTransactionApi::getDestinationTag() {
```

Comment:
Can you move that function nearby `setDestinationTag`, please?

Reply:
Of course !